exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTestWithArtifactsCreated.java

```diff
@@ -72,6 +78,17 @@ static void createInvalidArtifact(Path directory, String filename) throws IOExce
         TestSchema.class);
   }
 
+  static void checkIfServiceEnabled(TestKit testKit, String serviceName, int serviceId) {
```

Comment:
Should we create separate `TestKitTestUtils` and place it there or is this class good enough? If so, rename this class?

Reply:
It could live here, I think.